crates/tx5-connection/src/webrtc/go_pion.rs

```diff
@@ -107,15 +108,37 @@ async fn task(
     is_polite: bool,
     config: WebRtcConfig,
     send_buffer: usize,
+    evt_send: CloseSend<WebrtcEvt>,
+    cmd_send: CloseSend<Cmd>,
+    cmd_recv: CloseRecv<Cmd>,
+) -> Result<()> {
+    let config = config.to_go_buf()?;
+    let (peer, peer_evt) = tx5_go_pion::PeerConnection::new(config).await?;
+    task_inner(
+        is_polite,
+        send_buffer,
+        evt_send,
+        cmd_send,
+        cmd_recv,
+        peer,
+        peer_evt,
+    )
+    .await
+}
+
+async fn task_inner(
```

Comment:
Move logic into inner function so that we can access the `PeerConnection` in test